### PR TITLE
Remove ant functionality from publisher Endpoint

### PR DESCRIPTION
**Pull request by Krishanx92**
### Proposed changes in this pull request

Remove ant functionality from publisher Endpoint
Add theme provider to store app

### CODE REVIEW

**Comment by Krishanx92** [comment]
image
image

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/EndpointTableRows.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ */
+
+import React from 'react'
+import {Link} from 'react-router-dom'
+
+import {TableCell, TableRow} from 'material-ui/Table';
+import {ScopeValidation, resourceMethod, resourcePath} from '../../../data/ScopeValidation';
+import Dialog, {DialogActions, DialogContent, DialogContentText, DialogTitle} from 'material-ui/Dialog';
+import Button from 'material-ui/Button';
+import {Delete} from 'material-ui-icons';
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/EndpointTableRows.js]: import {Delete} from 'material-ui-icons/Delete'

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/EndpointTableRows.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                                </Button>
+                            </DialogActions>
+                        </Dialog></div>
+                </TableCell>
+            </TableRow>
+        );
+    }
+}
+
+export default EndpointTableRows
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/EndpointTableRows.js]: Missing ;

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/index.js [hunk 3]
```diff
@@ -74,99 +76,122 @@ export default class EndpointCreate extends Component {
...
-                            <Col offset={10}>
-                                <Row style={{marginBottom: "10px"}}>
-                                    <Col span={4}>Type</Col>
-                                    <Col span={8}>
-                                        <RadioGroup value={this.state.securityType}
+            <Grid container justify="center" spacing={0}>
+                <Grid item xs={10}>
+                    <Paper>
+                        <Grid item>
+                            <h4>
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/index.js]: Better to use Typography

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/index.js [hunk 4]
```diff
@@ -74,99 +76,122 @@ export default class EndpointCreate extends Component {
...
+                                Add new Global Endpoint
+                            </h4>
+                        </Grid>
+                        {/*<Grid item><form onSubmit={this.handleSubmit}>*/}<Grid>
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Create/index.js]: Remove comments

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Listing/index.js [hunk 5]
```diff
@@ -76,66 +91,72 @@ export default class EndpointsListing extends Component {
...
 
+    handleRequestCloseAddMenu() {
+        this.setState({openAddMenu: false});
+    };
+
+    handleRequestOpenAddMenu(event) {
+        this.setState({openAddMenu: true, anchorEl: event.currentTarget});
+    };
+
+
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Listing/index.js]: Remove unwanted space

**Pull request merged**